test/foundry/deploy/15_holograph_registry_tests.t.sol

```diff
@@ -0,0 +1,656 @@
+ * @dev This test verifies that the registry reverts with the message 'HOLOGRAPH: admin only function'
+ * when a random user (not the admin) attempts to alter the holograph slot.
+ * It simulates a random user calling the setHolograph function with a mock address.
+ * Refers to the hardhat tests with descriptions 'should fail to allow owner to alter _holographSlot' 
+ * and 'should fail to allow non-owner to alter _holographSlot'
+ */
+    function testAllowRandUserToAlterHolographSlotFail() public {
+        vm.prank(RandomAddress.randomAddress());
+        vm.expectRevert('HOLOGRAPH: admin only function');
+        holographRegistry.setHolograph(address(mockAddress));
+    }
+
+/*
+    getHolograph()
+*/
+
+/**
+ * @notice Test allowing an external contract to call the getHolograph function.
+ * @dev This test verifies that an external contract can successfully call the getHolograph function
+ * of the registry contract without any parameters.
+ * It simulates an external contract calling the getHolograph function in the registry.
+ * Refers to the hardhat test with the description 'Should allow external contract to call fn'
+ */
+    function testAllowExternalContractToCallFnGetHolograph() public {
+        bytes memory encodeSignature = abi.encodeWithSignature('getHolograph()');
+        mockExternalCall.callExternalFn(address(registryDeployedByScript), encodeSignature);           
+    }
+
+/*
+    setHToken()
+*/
+
+/**
+ * @notice Test allowing an admin to alter the hTokens mapping in the registry.
+ * @dev This test verifies that an admin can successfully update the hToken address for a specific chain ID
+ * in the registry with a new address.
+ * It simulates the admin calling the setHToken function with a valid chain ID and hToken address,
+ * and then checks if the registry correctly stores and returns the updated hToken address for the chain ID.
+ */
+    function testAllowAdminToAlterHTokens() public {
+        vm.prank(origin);
+        holographRegistry.setHToken(validChainId, hTokenAddress);
+        assertEq(holographRegistry.getHToken(validChainId), hTokenAddress);
+    }
+
+/**
+ * @notice Test the revert when a non-admin user tries to alter the hTokens mapping in the registry.
+ * @dev This test verifies that the registry reverts with the message 'HOLOGRAPH: admin only function'
+ * when a non-admin user attempts to alter the hToken address for a specific chain ID.
+ * It simulates a random user calling the setHToken function with a valid chain ID and hToken address.
+ * Refers to the hardhat tests with the descriptions 'should fail to allow owner to alter _hTokens' 
+ * and 'should fail to allow non-owner to alter _hTokens'
+ */
+    function testAllowNonOwnerToAlterHTokensReturn() public {
+        vm.prank(RandomAddress.randomAddress());
+        vm.expectRevert('HOLOGRAPH: admin only function');
+        holographRegistry.setHToken(validChainId, hTokenAddress);
+    }
+
+/*
+    getHToken()
```

Comment:
I see a lot of these blocks between `/* */` in this file such as 

```
/*
    setHToken()
*/
```

What's the reason for these?

Reply:
okay cool. Moving forward can we switch to a format like this to be consistent with how we're using dividers in our other contracts / tests?

```
    /* -------------------------------------------------------------------------- */
    /*                                 getHToken()                                */
    /* -------------------------------------------------------------------------- */

```

There's a very handy plugin for vscode we've been using that makes it easy https://marketplace.visualstudio.com/items?itemName=stackbreak.comment-divider

If you aren't using VS Code there might be other options available for your IDE. Just let us know